grow/pods/pods.py

```diff
@@ -290,6 +289,41 @@ def file_size(self, pod_path):
         path = self._normalize_path(pod_path)
         return self.storage.size(path)
 
+    def file_updated(self, pod_path):
```

Comment:
Maybe this should be `on_file_updated` to better indicate it's a callback function versus our other functions like `file_exists`?